chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -674,13 +672,13 @@ ingester.importFile.intoNewCollection = Import into new collection
 ingester.lookup.performing		= Performing Lookup…
 ingester.lookup.error			= An error occurred while performing lookup for this item.
 
-findPDF.searchingForAvailablePDFs = Searching for available PDFs…
+findPDF.searchingForAvailableFiles = Searching for available files…
 findPDF.checkingItems = Checking %1$S item;Checking %1$S items
-findPDF.pdfsAdded = %1$S PDF added;%1$S PDFs added
+findPDF.filesAdded = %1$S files added;%1$S files added
```

Comment:
`file` for the first one, but we should move this to Fluent while we're at it. (Technically these supported plural rules too, but I doubt anyone ever knew about it.)